compiler/release

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh -e
+
+# ----------------------------------------------------------------------
+# FPP release builder
+# ----------------------------------------------------------------------
+# To pass flags to sbt, set the environment variable
+# FPP_SBT_FLAGS
+# ----------------------------------------------------------------------
+
+DIRECTORY=`dirname $0`
+FAST_BIN="${DIRECTORY}/fast-fpp-`uname`-`uname -m`"
```

Comment:
Can we use the prefixes NATIVE_ and native- instead of fast? It's more precise. The native build should be faster, but all we can guarantee is that it's a native build.